engine_comparison/begin_experiment.sh

```diff
@@ -7,46 +7,58 @@
 DD=$(date +%d)
 MM=$(date +%m)
 INSTANCE_NAME="dispatcher-${DD}-${MM}"
-DISPATCHER_IMAGE_FAMILY=${DISPATCHER_IMAGE_FAMILY:-"ubuntu-1604-lts"} # Maybe container optimized
-export PROJECT_NAME="google.com:fuzz-comparisons"
+# DISPATCHER_IMAGE_FAMILY=${DISPATCHER_IMAGE_FAMILY:-"ubuntu-1604-lts"} # Maybe container optimized
+# export PROJECT_NAME="google.com:fuzz-comparisons"
 
 # These will frequently/usually be defined by the user
 export JOBS=${JOBS:-8}
 export N_ITERATIONS=${N_ITERATIONS:-5}
 
 # Define $BENCHMARKS
 if [[ $1 == 'all' ]]; then
-  for b in $(find ${SCRIPT_DIR}/*/build.sh -type f); do
+  for b in $(find ${SCRIPT_DIR}/../*/build.sh -type f); do
     BENCHMARKS="$BENCHMARKS $(basename $(dirname $b))"
   done
 #elif [[ $1 == 'other alias' ]]; do
 else
   BENCHMARKS=$(echo $1 | tr ',' ' ')
 fi
 
-# Create one gcloud instance for the dispatcher
-gcloud compute instances create $INSTANCE_NAME --image-family=$DISPATCHER_IMAGE_FAMILY --image-project=$PROJECT_NAME
+
+echo "Restarting gcloud instance. Instance should already be created (with gcloud_creator.sh)"
+# Start one gcloud instance for the dispatcher
+gcloud compute instances start $INSTANCE_NAME --zone=$GCLOUD_ZONE
+
+# Can't ssh immediately
+sleep 12 # Arbitrary choice of time, generally works
```

Comment:
This might work in practice, but is it possible to attempt a `gcloud compute ssh` and retry if it fails within a loop?  That seems like a more robust solution to me.